yt_idefix/fields.py

```diff
@@ -51,16 +49,52 @@ def setup_fluid_fields(self):
 
 class PlutoFields(BaseFields):
     known_other_fields = (
+        # PLUTO 4 standard variable names normalized in upper case,
+        # Referring to Tools/IDL/pload.pro in Pluto v4.2-patch2
+        # Ion fractions are not listed here
         # Each entry here is of the form
         # ( "name", ("units", ["fields", "to", "alias"], # "display_name")),
-        ("RHO", ("code_mass / code_length**3", ["density", "rho"], None)),
-        ("VX1", ("code_length/code_time", ["vel_x"], None)),
-        ("VX2", ("code_length/code_time", ["vel_y"], None)),
-        ("VX3", ("code_length/code_time", ["vel_z"], None)),
-        ("PRS", ("code_pressure", ["prs", "pres", "pressure"], None)),
+        # alias should be set with yt-standard-name or pluto-macro-name
+        ("RHO", ("code_density", ["density"], None)),
+        ("VX1", ("code_velocity", ["velocity_x"], None)),
+        ("VX2", ("code_velocity", ["velocity_y"], None)),
+        ("VX3", ("code_velocity", ["velocity_z"], None)),
+        ("PRS", ("code_pressure", ["pressure"], None)),
+        ("AX1", ("code_magnetic", [], None)),
+        ("AX2", ("code_magnetic", [], None)),
+        ("AX3", ("code_magnetic", [], None)),
         ("BX1", ("code_magnetic", [], None)),
         ("BX2", ("code_magnetic", [], None)),
         ("BX3", ("code_magnetic", [], None)),
+        ("EX1", ("code_magnetic/c", [], None)),
+        ("EX2", ("code_magnetic/c", [], None)),
+        ("EX3", ("code_magnetic/c", [], None)),
+        ("BX1S", ("code_magnetic", ["BX1s"], None)),
+        ("BX2S", ("code_magnetic", ["BX2s"], None)),
+        ("BX3S", ("code_magnetic", ["BX3s"], None)),
+        ("EX1S", ("code_magnetic/c", ["EX1s"], None)),
+        ("EX2S", ("code_magnetic/c", ["EX2s"], None)),
+        ("EX3S", ("code_magnetic/c", ["EX3s"], None)),
+        ("CRG", ("", ["code_magnetic/code_length/c"], None)),
+        ("PSI_GLM", ("", [], None)),
+        ("PHI_GLM", ("", [], None)),
+        ("RHO_D", ("code_density", [], None)),
+        ("VX1_D", ("code_velocity", [], None)),
+        ("VX2_D", ("code_velocity", [], None)),
+        ("VX3_D", ("code_velocity", [], None)),
```

Comment:
out of curiosity, does the `_D` suffix mean "dust" ?

Reply:
yes

Reply:
do you happen to know wether these represent a pressure less fluid or if it's the result of particle deposition ?

Reply:
I just checked it. I think they represent a pressure less fluid, which is introduced in a related file `dust_fluid.c` but not written in the user guide.

Reply:
BTW, there is another type of dust as a kind of particle mentioned in the user guide. This module is still under construction and not public.

Reply:
Good to know ! Thank you very much.
I'll make a last review later today and probably merge + release